cttportfolio.css

```
@@ -467,4 +540,194 @@ footer{
 	max-height:100%;
 	border-radius:50%;
 }
-/*======================================== MEDIA QUERIES SECTION ========================================*/
\ No newline at end of file
+/*======================================== MEDIA QUERIES SECTION ========================================*/
+@media only screen and (max-width: 1100px){
+	.navsublinks{
+		display:none;
+	}
+	.navlinks .nav_dropdown{
+		color:rgb(248,248,255);
+		display:inline-block;
+		font-size:3rem;
+		transform: scale(1,1);
+		border:1px solid rgb(248,248,255);
+		padding:8px 5px;
+		padding-bottom:0px;
```

You don't need units when using `0`.
```suggestion
		padding-bottom:0;
```